guides/default-technology-stack.md

```diff
@@ -47,15 +47,11 @@ Community support for Azure use in general can also be gained from the community
 
 ### GOV.UK PaaS
 
-We are currently exploring usage of [GOV.UK PaaS](https://www.cloud.service.gov.uk/) as a secondary offering, particularly for [12 factor apps](https://12factor.net/).
+We are currently exploring usage of [GOV.UK Platform as a Service](https://www.cloud.service.gov.uk/) as a secondary offering for applications requiring less customisation than infrastructure as a service like Azure.
```

Comment:
"a secondary offering for applications requiring less customisation than provided by a full infrastructure-as-a-service platform such as Azure"

Think that's clearer